site/content/docs/0.0/getting-started/accessibility.md

```diff
@@ -102,6 +102,6 @@ When writing a paragraph, it is commonly admitted that a line should have 80 cha
 - ["HTML Codesniffer" bookmarklet for identifying accessibility issues](https://github.com/squizlabs/HTML_CodeSniffer)
 - [Microsoft Accessibility Insights](https://accessibilityinsights.io/)
 - [Deque Axe testing tools](https://www.deque.com/axe/)
-- [Orange accessibility guidelines](http://a11y-guidelines.orange.com/)
+- [Orange accessibility guidelines](http://a11y-guidelines.orange.com/en/)
```

Comment:
why defining the language to "en" ? There is a favorite language detection on this website to display the preferred language. Isn't it better to let the visitor navigate on is preferred language ?

Reply:
@hannahiss will be able to add more elements to my answer, but based on the accessibility audit, the language of the label (link) should be the same as the language used in the target website to prevent language switching.
IMHO, I prefer how it was handled before with favorite language detection too.

Reply:
@julien-deramond  @vprothais yes, the problem is that we have to mention if the link leads to a website in a different language for accessibility. The user can still switch to french if he prefers it...

Reply:
Ok thank you for the explanation. I understand the point. I find it a bit weird, but I understand it could be disturbing to arrive on a website with another language. Mentioning the language in the link would be tricky when it aim a website with a language detection since we don't know what will be the result for the user.